app/views/guides/_form.html.erb

```diff
@@ -3,34 +3,40 @@
     <fieldset class='meta'>
       <legend>Meta</legend>
       <div class='form-group'>
-        <%= f.label :slug %>
+        <%= f.label(:slug) %>
```

Comment:
I think the convention is to omit parenthesis on method calls in views unless they're really needed.